src/base/loaders/I3DMLoaderBase.js

```diff
@@ -84,13 +84,13 @@ export class I3DMLoaderBase extends LoaderBase {
 
 		} else {
 
-			const externalUri = this.resolveExternalURL( arrayToString( bodyBytes ) );
-			promise = fetch( externalUri, this.fetchOptions )
+			this.workingPath = this.resolveExternalURL( arrayToString( bodyBytes ) );
+			promise = fetch( this.workingPath, this.fetchOptions )
```

Comment:
Working path shouldn't be set to the file URL - it should be set to the path root. Ie we should remove the file name from the path if we're going to call it a working path. Then the GLTFLoader can use it directly.

Reply:
If I'm following this 
https://github.com/NASA-AMMOS/3DTilesRendererJS/pull/734#discussion_r1749055695

Should I just revert this part and come back to `externalUri` ?

Reply:
Yeah - hopefully the answer in https://github.com/NASA-AMMOS/3DTilesRendererJS/pull/734/files#r1751888696 clarifies things. Let me know if I need to rephrase it.